### PR TITLE
Readded victory create methods and imports

### PR DESCRIPTION
**Pull request by Alek99**
### All Submissions:

- [ ] Have you followed the guidelines stated in [CONTRIBUTING.md](https://github.com/reflex-dev/reflex/blob/main/CONTRIBUTING.md) file?
- [ ] Have you checked to ensure there aren't any other open [Pull Requests](https://github.com/reflex-dev/reflex/pulls ) for the desired changed?

### Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### New Feature Submission:

- [ ] Does your submission pass the tests? 
- [ ] Have you linted your code locally prior to submission?

### Changes To Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### **After** these steps, you're ready to open a pull request.

    a. Give a descriptive title to your PR.

    b. Describe your changes.

    c. Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).

### CODE REVIEW

File: reflex/components/__init__.py [hunk 1]
```diff
@@ -100,7 +100,6 @@
 editable_input = EditableInput.create
 editable_preview = EditablePreview.create
 editable_textarea = EditableTextarea.create
-editor = Editor.create
```
Comment (masenf) [inline reflex/components/__init__.py:103]: still want to keep the new editor exposed.

Comment (Alek99) [reply reflex/components/__init__.py:103]: Good catch my bad

**Review by masenf — CHANGES_REQUESTED** [review]
```
AttributeError: module 'reflex' has no attribute 'data'
```

😔

**Comment by Alek99** [comment]
Ok let me actually test this myself I just sent this pr without looking

**Comment by Alek99** [comment]
Ok tested this myself with reflex web when victory was in it

**Review by masenf — APPROVED** [review]

**Pull request merged**